src/services/fileServices/MdPageServices/MediaFileService.js

```diff
@@ -21,7 +21,18 @@ class MediaFileService {
 
   async create(sessionData, { fileName, directoryName, content }) {
     this.mediaNameChecks({ directoryName, fileName })
-    const sanitizedContent = await validateAndSanitizeFileUpload(content)
+
+    const [schema, fileContent] = content.split(",")
+    const fileBuffer = Buffer.from(fileContent, "base64")
+
+    // Scan file for virus - cloudmersive API
+    const virusScanRes = await scanFileForVirus(fileBuffer)
```

Comment:
should we log intermediate results for backtracing later since this is a "new" feature

Reply:
I mean to log the return data from Cloudmersive api call where the file is ok or not